drivers/SmartThings/matter-pump/src/init.lua

```diff
@@ -176,7 +176,8 @@ end
 
 local function level_attr_handler(driver, device, ib, response)
   if ib.data.value ~= nil then
-    local level = math.floor((ib.data.value / 254.0 * 100) + 0.5)
+    local level = math.floor((ib.data.value / 200 * 100) + 0.5)

```

Comment:
Could you define these magic numbers at the top of the file?

```
-- Per matter spec, the pump level is in steps of 0.5% and the max level value is 200. Anything above is considered 100%
local MAX_PUMP_ATTR_LEVEL = 200
local MAX_LEVEL = 100
```
And then change this to something like
```suggestion
    local level = math.floor((ib.data.value / MAX_PUMP_ATTR_LEVEL * MAX_LEVEL) + 0.5)
    level = math.max(level, MAX_LEVEL)
```

Reply:
done :+1: